README.md

```diff
@@ -2,31 +2,62 @@
 
 [![Build Status](https://github.com/bioio-devs/bioio-tifffile/actions/workflows/ci.yml/badge.svg)](https://github.com/bioio-devs/bioio-tifffile/actions)
 [![Documentation](https://github.com/bioio-devs/bioio-tifffile/actions/workflows/docs.yml/badge.svg)](https://bioio-devs.github.io/bioio-tifffile)
+[![PyPI version](https://badge.fury.io/py/bioio-tifffile.svg)](https://badge.fury.io/py/bioio-tifffile)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Python 3.9+](https://img.shields.io/badge/python-3.9,3.10,3.11-blue.svg)](https://www.python.org/downloads/release/python-390/)
 
-A BioIO reader plugin for reading TIFFs using Tifffile
+A BioIO reader plugin for reading TIFFs using `tifffile`
 
 ---
 
+
+## Documentation
+
+[See the full documentation on our GitHub pages site](https://bioio-devs.github.io/bioio/OVERVIEW.html) - the generic use and installation instructions there will work for this package.
+
 ## Installation
 
 **Stable Release:** `pip install bioio-tifffile`<br>
 **Development Head:** `pip install git+https://github.com/bioio-devs/bioio-tifffile.git`
 
-## Quickstart
+## Example Usage (see full documentation for more examples)
 
-```python
-from bioio_tifffile import Reader 
+Install bioio-tifffile alongside bioio:
 
-r = Reader("my-image.tiff")
-r.dims
+`pip install bioio bioio-tifffile`
+
+```python
+from bioio import BioImage
+
+# Get a BioImage object
+img = BioImage("my_file.tiff")  # selects the first scene found
```

Comment:
is this the type of documentation you want for a reader?  this is the more general bioio usage but doesn't really say anything about this reader.  I guess I would suggest either show the public api of this reader itself, or at least show `img = BioImage("myfile.tiff", reader=bioio_tifffile.Reader)`

Reply:
I wonder if specifying the reader there would make people think they need to do that, I'll add it but mention it is optional